### PR TITLE
Partial (1/2) spanish translation ch08-01-vectors.md

### PR DESCRIPTION
**Pull request by jossue33**
Line: 1-158
Number of words: 1018
Part: 1/2

### CODE REVIEW

File: second-edition/src/ch08-01-vectors.md [hunk 1]
```diff
@@ -129,33 +129,33 @@ let does_not_exist = &v[100];
...
-ownership and borrowing rules (covered in Chapter 4) to ensure this reference
-and any other references to the contents of the vector remain valid. Recall the
-rule that states we can’t have mutable and immutable references in the same
-scope. That rule applies in Listing 8-7 where we hold an immutable reference to
-the first element in a vector and try to add an element to the end, which won’t
-work:
+<span class="caption">Listando 8-6: Intentando acceder al elemento en el índice
+100 en un vector que contiene 5 elementos </ span>
+
+Cuando ejectures este código, el primer método `[]` causará `panic!` por que este
```
Comment (thecodix) [inline second-edition/src/ch08-01-vectors.md]: _...Este método se usa mejor cuando quiere su programa para considerar un intento de acceder a un elemento más allá del final del vector sería un error fatal que bloquea el programa..._

Please consider revising the translated code, here the sentence can be better phrased (is hard to understand what is referring to)

File: second-edition/src/ch08-01-vectors.md [hunk 2]
```diff
@@ -107,20 +107,20 @@ let third: &i32 = &v[2];
 let third: Option<&i32> = v.get(2);
 ```
 
-<span class="caption">Listing 8-5: Using indexing syntax or the `get` method to
-access an item in a vector</span>
+<span class="caption">Listando 8-5: Usando la sintaxis de indexación o el método `get` para
```
Comment (thecodix) [inline second-edition/src/ch08-01-vectors.md]: 'indexacion' should be changed by 'indices'

File: second-edition/src/ch08-01-vectors.md [hunk 3]
```diff
@@ -82,23 +82,23 @@ annotated in Listing 8-4:
 } // <- v goes out of scope and is freed here
 ```
 
-<span class="caption">Listing 8-4: Showing where the vector and its elements
-are dropped</span>
+<span class="caption">Listando 8-4: Mostrando donde el vector y sus elementos
+son dejados caer</span>
```
Comment (thecodix) [inline second-edition/src/ch08-01-vectors.md]: 'dejados caer' -> 'descartados' 

Please consider revising where else is this word used

File: second-edition/src/ch08-01-vectors.md [hunk 4]
```diff
@@ -60,18 +60,18 @@ v.push(7);
...
-<span class="caption">Listing 8-3: Using the `push` method to add values to a
+<span class="caption">Lisando 8-3: Usando el método `push` para agregar valores a un
 vector</span>
 
-As with any variable, as discussed in Chapter 3, if we want to be able to
-change its value, we need to make it mutable using the `mut` keyword. The
-numbers we place inside are all of type `i32`, and Rust infers this from the
-data, so we don’t need the `Vec<i32>` annotation.
+Como con cualquier variable, como se discutió en el Capítulo 3, si queremos ser capaces de
+cambiar su valor, tenemos que hacerlo mutable usando la palabra clave `mut`. los
```
Comment (thecodix) [inline second-edition/src/ch08-01-vectors.md]: 'los' is duplicated

File: second-edition/src/ch08-01-vectors.md [hunk 5]
```diff
@@ -1,55 +1,55 @@
-## Vectors Store Lists of Values
+## ## Listas de valores guardando vectores
```
Comment (thecodix) [inline second-edition/src/ch08-01-vectors.md]: Los Vectores Almacenan Listas de Valores

**Comment by jossue33** [comment]
@thecodix todo listo.

**Comment by thecodix** [comment]
machine translated

**Pull request merged**